.github/workflows/ci.yml

```diff
@@ -43,4 +43,23 @@ jobs:
         run: |
           pytest --cov=eventio --cov-report=xml
 
+      - name: Run examples
```

Comment:
probably should set `MPLBACKEND=agg` to not create windows in the CI for the plots

Reply:
I do not see a change in adding it, but no strong feelings

Reply:
done